modal-go/cls.go

```diff
@@ -22,7 +22,9 @@ type clsServiceImpl struct{ client *Client }
 // ClsWithOptionsParams represents runtime options for a Modal Cls.
 type ClsWithOptionsParams struct {
 	CPU              *float64
+	CPUMax           *float64
```

Comment:
If we want to keep the same phrasing as https://modal.com/docs/guide/resources#reserving-cpu-and-memory, "CPU Limits" then:

```suggestion
	CPULimit           *float64
```

Same comment for memory.